### DISCUSSION TITLE
Looking to enable larger lane hitboxes in mania on steamdeck

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#28334] [Q&A] [answered]
Hello,

on steamdeck, is it possible to enable the oversized lane hitboxes for touch controls in mania similar to mobile? currently i am forced to touch the narrow lanes normally seen on desktop, but obviously this isnt ideal. the game natively detects the touch screen (standard enables the touch mod and i can play chords in mania) so i was surprised i couldnt enable the larger touch zones.

**Comment by peppy** [comment, marked as answer]
In the latest release touch works across the whole screen regardless of platform.
